README.md

```diff
@@ -1,17 +1,42 @@
-# Baremetal Deploy
+# OpenShift Baremetal Deploy
 
-Deployment artifacts for Bare Metal OpenShift KNI clusters
+This repository is intended to store resources and deployment artifacts
+for [bare metal](https://github.com/metal3-io/metal3-docs/blob/master/design/bare-metal-style-guide.md)
+OpenShift KNI clusters (currently OpenShift 4.3)
+
+It also contains optional features focused on low-latency workloads, NFV workloads, etc.
+
+## Installation artifacts
 
-## Current Artifacts
 * [Installation Steps](install-steps.md)
 * [Using Baremetal-prep.sh](baremetal-prep.md)
 
-## To Be Added
-* Additional step for disconnected install
-* Installation Troubleshooting
-* Installation scripts
-* Various feature yamls/manifest
+## Optional features
+
+* [CPU Manager](features/cpumanager/). CPU Manager manages groups of CPUs and constrains workloads
+to specific CPUs.
+* [PTP](features/ptp). This operator manages cluster wide Precision Time Protocol (PTP) configuration.
+* [SCTP](features/sctp). These assets enables Stream Control Transmission Protocol (SCTP) in the RHCOS
+worker nodes.
+* [SRIOV](features/sriov). The SR-IOV Network Operator creates and manages the components of the SR-IOV stack.
+
+## Performance tuning
+
+The [Performance Tuning](features/performance) folder contains some assets intended to improve performance such as:
+
+* Huge Pages
+* Topology Manager
+* Real time kernel (including a new `worker-rt` Kubernetes/OpenShift node role)
+
+Those assets are applied by using the [Node Tuning](https://github.com/openshift/cluster-node-tuning-operator)
+and the [Machine Config](https://github.com/openshift-kni/baremetal-deploy/blob/master/features/sriov/deploy.sh) operators.
```

Comment:
Machine Config operator should link to https://github.com/openshift/machine-config-operator.
The deploy script is a moving target at the moment, see #59

Reply:
I've rephrased the explanation and fixed the link (bad copypasta!)